opensrp-reveal/src/test/java/org/smartregister/reveal/presenter/ListTaskPresenterTest.java

```diff
@@ -103,8 +154,10 @@ public void setUp() {
         org.smartregister.Context.bindtypes = new ArrayList<>();
         listTaskPresenter = new ListTaskPresenter(listTaskView, drawerPresenter);
         Whitebox.setInternalState(listTaskPresenter, "listTaskInteractor", listTaskInteractor);
+        Whitebox.setInternalState(listTaskPresenter, "jsonFormUtils", jsonFormUtils);
         prefsUtil.setCurrentPlanId(planId);
         prefsUtil.setCurrentOperationalArea(operationalArea);
+        when(listTaskView.getContext()).thenReturn(Robolectric.buildActivity(Activity.class).resume().get());
```

Comment:
This may not be efficient why not pass `RuntimeEnvironment.application` or create field activity that is initialized only once.

With this approach of anonymous variable an activity is always created and resumed when `getContext` is called